common/model/requesting.ts

```diff
@@ -0,0 +1,32 @@
+import { Identifier, PhysicalLocation } from './catalogue';
+
+type PickupLocation = {
+  id: string;
+  label: string;
+  type: 'LocationDescription';
+};
+
+type RequestStatus = {
+  id: string;
+  label: string;
+  type: 'RequestStatus';
+};
+
+type RequestItem = {
+  item: {
+    id?: string;
```

Comment:
The optional fields here, are they ever optional. Feels odd with `id` but I know we sometimes don't mint IDs for things.